src/ts/core/features/vim-mode/index.ts

```diff
@@ -13,7 +15,7 @@ import {HintCommands} from 'src/core/features/vim-mode/commands/hint-commands'
 
 export const config: Feature = {
     id: 'block_navigation_mode',
-    name: 'Vim-like Block Navigation (requires refresh)',
+    name: 'Vim-like Block Navigation (refresh to disable)',
```

Comment:
it should be possible to add disable on toggle as-well, right? theoretically we just need to collect and call a bunch of disconnect functions, right?